### DISCUSSION TITLE
ST7920, how is

### DISCUSSION THREAD

**Discussion by tomjennings** [olikraus/u8g2#2084] [General]
I have a 144 x 32 LCD driven by a ST7920 chip. (Tight physical-height requirement, displays is 36mm tall, all the room I have). I've used u8g before, but never had this particular problem: how to specify/set display size in pixels.

There's `getDisplayHeight()` and `getDisplayWidth()` methods; are there corresponding `set` methods?

Or should I just choose a constructor for a differently geometry, eg. 160 x 32, and restrict my horizontal addresses to 144? Eg. `U8G2_ST7920_160X32_F_6800(rotation, d0, d1, d2, d3, d4, d5, d6, d7, enable, cs, dc [, reset]) [full framebuffer, size = 640 bytes]`?

SORRY newbie error: somehow clicked POST with incomplete subject line. No way to edit subject nor delete post.

**Comment by olikraus** [comment]
probably you need a new constructor.
Issue #2091 created
